modules/ROOT/pages/keycloak/authenticating.adoc

```diff
@@ -115,13 +115,73 @@ authService.login().then(() => {
 IAuthService authService = MobileCore.Instance.GetService<IAuthService>();
 
 var authOptions = DependencyService.Get<IAuthenticateOptionsProvider>().GetOptions();
-authService.Authenticate(authOptions).ContinueWith(user =>
-{
-    // Login success
+var user = await authService.Authenticate(authOptions);
+----
+****
+
+== Refreshing the authentication token
+
+Every time a user is authenticated, both an access and a refresh token get generated. They both have a lifespan that can be configured on the authentication server.
+If the refresh token is still valid, the authentication token can be refreshed even if it is currently expired.
```

Comment:
If the refresh token is still valid, the authentication token can be refreshed even if the authentication token has expired.